### PR TITLE
#53485 wp date code paths

### PR DESCRIPTION
**Pull request by pbearne**
added missing coverage paths  to wp_date

Trac ticket: https://core.trac.wordpress.org/ticket/53485

### CODE REVIEW

File: tests/phpunit/tests/date/wpDate.php [hunk 1]
```diff
@@ -61,4 +61,55 @@ public function test_should_keep_localized_slashes() {
...
 	}
+
+	/**
+	 * Test if no timestamp provides
+	 *
+	 * @ticket 53485
+	 */
+	public function test_no_timestamp() {
+
+		$this->assertSame( (string) strtotime( 'now' ), wp_date( 'U' ) );
```
Comment (Rarst) [inline tests/phpunit/tests/date/wpDate.php:72]: `time()` for current timestamp? `strtotime()` is a mess. :)

File: tests/phpunit/tests/date/wpDate.php [hunk 2]
```diff
@@ -61,4 +61,55 @@ public function test_should_keep_localized_slashes() {
...
+	}
+
+	/**
+	 * Test if format is set to F weekday_abbrev
+	 *
+	 * @ticket 53485
+	 */
+	public function test_format_F() {
```
Comment (Rarst) [inline tests/phpunit/tests/date/wpDate.php:80]: Test says F, code tests D.

File: tests/phpunit/tests/date/wpDate.php [hunk 3]
```diff
@@ -61,4 +61,55 @@ public function test_should_keep_localized_slashes() {
...
+	 */
+	public function test_format_M() {
+		$utc      = new DateTimeZone( 'UTC' );
+		$datetime = new DateTimeImmutable( '2019-10-17', $utc );
+
+		$this->assertSame( 'Oct', wp_date( 'M', $datetime->getTimestamp(), $utc ) );
+	}
+
+	/**
+	 * Test if format is set to M month_abbrev
```
Comment (Rarst) [inline tests/phpunit/tests/date/wpDate.php:100]: Description from another test.

File: tests/phpunit/tests/date/wpDate.php [hunk 4]
```diff
@@ -61,4 +61,55 @@ public function test_should_keep_localized_slashes() {
...
+	 *
+	 * @ticket 53485
+	 */
+	public function test_wp_date_filter() {
```
Comment (Rarst) [inline tests/phpunit/tests/date/wpDate.php:104]: Hm hm, do we need an explicit test that filter works for a specific function? Is there a precedent in tests for doing this every time?

File: tests/phpunit/tests/date/wpDate.php (hunk 1 above)
Comment (pbearne) [reply tests/phpunit/tests/date/wpDate.php]: done

File: tests/phpunit/tests/date/wpDate.php (hunk 3 above)
Comment (pbearne) [reply tests/phpunit/tests/date/wpDate.php]: fixed

File: tests/phpunit/tests/date/wpDate.php (hunk 4 above)
Comment (pbearne) [reply tests/phpunit/tests/date/wpDate.php:177]: yes this makes sure the filter is there

**Comment by peterwilsoncc** [comment]
Superseded by PR #2643

**Pull request closed without merging**